### DISCUSSION TITLE
2012/01/28/%E8%B1%86%E8%B1%86%E7%9A%842012%E6%98%A5%E8%8A%82

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#353] [Announcements]
# 2012/01/28/%E8%B1%86%E8%B1%86%E7%9A%842012%E6%98%A5%E8%8A%82

豆豆趴在大床的正中：面向左侧，右脸紧贴着床单，嘴巴在挤压下嘟成了一小团；两条腿蜷在肚子两侧，屁股撅撅着，把被子顶了个大包；豆豆右手向屁股的方向耷拉着，左手被我紧紧攥着快两个小时了。我斜躺在豆豆边上，看他安详的一呼一吸。要不是春节长假，我恐怕还很难找出这么长的时间看着他睡午觉。随着年纪变大，我明显察觉时间开始加速流逝。仿佛就在前天，老婆打来电话说她似乎有了；昨天我还在产房外焦急的等待；今天豆豆就已经开始呀呀学语了。即便是现在正握着他的小手，仍然感到有些惶恐，生怕还没享受够他带来的快乐，再一觉醒来，豆豆就已经长大要远走高飞。

https://blog.qizhen.xyz/2012/01/28/%E8%B1%86%E8%B1%86%E7%9A%842012%E6%98%A5%E8%8A%82

**Comment by ruanqizhen** [comment]
之前博客下的留言：

xuqinglong
2012年02月29日 22:34 编辑
刚翻出来看你的博客，读您的书很亲切。
小朋友很可爱!

回复
